docs/cli/auth/admin.md

```diff
@@ -104,3 +108,143 @@ function App() {
+        switch result {
+        case .success(let data):
+            print("Response Body: \(String(decoding: data, as: UTF8.self))")
+        case .failure(let error):
+            if case let .httpStatusError(statusCode, response) = error,
+                let awsResponse = response as? AWSHTTPURLResponse,
+                let responseBody = awsResponse.body
+            {
+                print("StatusCode: \(statusCode) Response Body: \(String(decoding: responseBody, as: UTF8.self))")
+            }
+        }
+    }
+}
+
+addToGroup(username: "richard", groupName:  "Editors")
+```
+
+4. Use the following to list the users in the Group.
+
+```swift
+func listEditors(groupName: String, limit: Int, nextToken: String? = nil) {
+    let path = "/listUsersInGroup"
+    var query = ["groupname": groupName,
+                  "limit": String(limit)]
+    if let nextToken = nextToken {
+        query["token"] = nextToken
+    }
+    
+    let request = RESTRequest(path: path, queryParameters: query, body: nil)
+    Amplify.API.get(request: request) { result in
+        switch result {
+        case .success(let data):
+            print("Response Body: \(String(decoding: data, as: UTF8.self))")
+        case .failure(let error):
+            if case let .httpStatusError(statusCode, response) = error,
+                let awsResponse = response as? AWSHTTPURLResponse,
+                let responseBody = awsResponse.body
+            {
+                print("StatusCode: \(statusCode) Response Body: \(String(decoding: responseBody, as: UTF8.self))")
+            }
+        }
+    }
+}
+
+listEditors(groupName: "Editors", limit: 10)
+```
+
+**Note: Cognito User Pool with HostedUI** 
+
+The Admin Queries API configuration in **amplifyconfiguration.json** will have the endpoint's authorization type set to `AMAZON_COGNITO_USER_POOLS`. With this authorization type, `Amplify.API` will perform the request with the access token. However, when using HostedUI, the app may get unauthorized responses despite being signed in, and will require using the ID Token. Set the authorizationType to "NONE" and add a custom interceptor to return the ID Token. 
+
+```json
+{
+    "awsAPIPlugin": {
+        "[YOUR-RESTENDPOINT-NAME]": {
+            "endpointType": "REST",
+            "endpoint": "[YOUR-REST-ENDPOINT]",
+            "region": "[REGION]",
+            "authorizationType": "NONE"
+        }
+    }
+}
+```
+<amplify-callout warning>
+
+If you perform additional updates to your resources using Amplify CLI, the authorizationType will be reverted back to `AMAZON_COGNITO_USER_POOLS`. Make sure to update this back to `NONE`.
+
+</amplify-callout>
+
+Add a custom interceptor to the API
+```swift
+try Amplify.configure()
+try Amplify.API.add(interceptor: MyCustomInterceptor(), for: "[YOUR-RESTENDPOINT-NAME]")
+```
+
+Set up the custom interceptor to return the ID token for the request.
+
+```swift
+class MyCustomInterceptor: URLRequestInterceptor {
```

Comment:
which library should be imported for URLRequestInterceptor?  It would be useful to include import statements in case people are not clear.

Reply:
URLRequestInterceptor comes from AWSAPIPlugin, which is imported differently based on the dependency manager, ie. Cocoapods is `import AmplifyPlugins` and Swift Package Manager is `import AWSAPIPlugin`. I'll mention it here